### PR TITLE
videoNow bid adapter

### PR DESCRIPTION
**Pull request by sdbaron**
## Type of change
- [x ] New bidder adapter   

## Description of change
new bidder adapter for videoNow

- test parameters for validating bids
```
   {
      code: 'banner-div',
      mediaTypes: {
        banner: {
          sizes: [[640, 480], [336, 280]]
        }
      },
      bids: [{
         bidder: 'videonow',
         params: {
           pId: 1,
           placementId: 'div-001'
         }
       }]
   }
```

- contact email of the adapter’s maintainer
- [x ] official adapter submission

### CODE REVIEW

**Comment by sdbaron** [comment]
> Hi @sdbaron,
> I am getting `404 Not Found` error when hitting the url `https://bidder.videonow.ru/prebid?profile_id=1`. I am making request from `India` with the following request payload: `{"id":"2b9273013ea69d","sizes":[[640,480],[336,280]],"cur":"RUB","placementId":"div-001","ref":"http://localhost:9999/integrationExamples/gpt/hello_world.html?pbjs_debug=true"}`
> 
> Please check at your end.

Hi!
Indeed, we had some problems with this URL, but now it should work correctly.

File: modules/videoNowBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+      ttl: ttl || TTL_SECONDS,
+      ad: code,
+      nurl,
+      renderer: {
+        url: nurl,
+        render: function() {
+          const d = window.document
+          const el = placementId && d.getElementById(placementId)
+          if (!el) {
+            console.error(`bidAdapter ${BIDDER_CODE}: ${placementId} not found`)
```
Comment (sumit116) [inline modules/videoNowBidAdapter.js]: can we use `utils.logError()` here?

File: modules/videoNowBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  buildRequests,
+  interpretResponse,
+  getUserSyncs,
+  onTimeout: function(timeoutData) {},
+  onBidWon
+}
+
+registerBidder(spec);
+
+function reInitScripts(container, shouldReInitInlineScripts = true, delay = 10) {
```
Comment (sumit116) [inline modules/videoNowBidAdapter.js]: Can you please explain what does this function do and why do we need it in the adapter?

File: test/spec/modules/videoNowBidAdapter_spec.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
+import { expect } from 'chai';
```
Comment (sumit116) [inline test/spec/modules/videoNowBidAdapter_spec.js]: unit test coverage is very low.

**Comment by sumit116** [comment]
Hi @sdbaron, could you work on my feedback or do you have any questions?

**Comment by sdbaron** [comment]
Hi @sumit116, I had a vacation for a week, but now I’ve returned to work and will make corrections in a couple of days.

**Comment by sumit116** [comment]
Hi @sdbaron, I still didn't recieve the response from `https://bidder.videonow.ru/prebid?profile_id=1`. May be you need to check on that. Apart from that, the request object looks fine and unit test coverage is now 100%, so approving the PR.

**Comment by sdbaron** [comment]
@sumit116 https://github.com/prebid/prebid.github.io/pull/1506

**Pull request merged**